app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -105,12 +105,12 @@ def self.sync_workers
       result
     end
 
-    def self.thin_pid_file(port)
-      Rails.root.join("tmp/pids/thin.#{port}.pid")
+    def self.pid_file(port)
+      Rails.root.join("tmp/pids/puma.#{port}.pid")
```

Comment:
Note, you might have to create an accessor method in the module to access the constant like used above (there are other ways so anything that gets the right constant is fine by me):

```
   def self.binding_address
       BINDING_ADDRESS
     end
```